### PR TITLE
Adding Product Recommendations Specific Cookies

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) adds information about the cookies installed with the product recommendations module.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/stores/cookie-reference.html

### CODE REVIEW

File: src/stores/cookie-reference.md [hunk 1]
```diff
@@ -285,5 +285,17 @@ JavaScript Library: `ga.js`
...
+{:.ee-only}
+## Product Recommendations Cookies
+
+The following cookies are used by Product Recommendations for Magento Commerce customers. These cookies are installed with the [DataServices module](https://devdocs.magento.com/recommendations/install-configure.html).
+
+- `mg_dnt`: Allows you to [restrict Magento data collection](https://devdocs.magento.com/recommendations/setting-cookie.html) if you have custom code to manage cookie consent on your site.
+- `user_allowed_save_cookie`: Used for Magento's [cookie restriction mode]({% link stores/compliance-cookie-restriction-mode.md %}).
+- `authentication_flag`: Tracks a shopper's sign in and sign out status.
+- `dataservices_customer_id`: Tracks a shopper's sign in and sign out actions.
+- `dataservices_cart_id`: Used with a shopper's cart actions.
```
Comment (misha-kotov) [inline src/stores/cookie-reference.md:297]:
```suggestion
- `dataservices_cart_id`: Tracks a shopper's shopping cart.
```
let's be consistent

File: src/stores/cookie-reference.md [hunk 2]
```diff
@@ -285,5 +285,17 @@ JavaScript Library: `ga.js`
...
 
+{:.ee-only}
+## Product Recommendations Cookies
+
+The following cookies are used by Product Recommendations for Magento Commerce customers. These cookies are installed with the [DataServices module](https://devdocs.magento.com/recommendations/install-configure.html).
+
+- `mg_dnt`: Allows you to [restrict Magento data collection](https://devdocs.magento.com/recommendations/setting-cookie.html) if you have custom code to manage cookie consent on your site.
+- `user_allowed_save_cookie`: Used for Magento's [cookie restriction mode]({% link stores/compliance-cookie-restriction-mode.md %}).
+- `authentication_flag`: Tracks a shopper's sign in and sign out status.
+- `dataservices_customer_id`: Tracks a shopper's sign in and sign out actions.
```
Comment (misha-kotov) [inline src/stores/cookie-reference.md:296]: can we add something like "counter to its name, does not contain the customer id" ?

**Review by misha-kotov — APPROVED** [review]

File: src/stores/cookie-reference.md (hunk 1 above)
Comment (rossbrandon) [reply src/stores/cookie-reference.md:297]: I thought we were avoiding the word "Track" or "Tracking"

**Review by rossbrandon — APPROVED** [review]

File: src/stores/cookie-reference.md (hunk 1 above)
Comment (erikmarr) [reply src/stores/cookie-reference.md:297]: @misha-kotov thoughts on Ross's question?

Comment (misha-kotov) [reply src/stores/cookie-reference.md:297]: yeah, agree, we should find an alternative. any leads from existing cookie docs?

Comment (leslietilling) [reply src/stores/cookie-reference.md:297]: Maybe "identifies" ?

File: src/stores/cookie-reference.md [hunk 3]
```diff
@@ -285,5 +285,17 @@ JavaScript Library: `ga.js`
 - `_utmz`: Saves the traffic source or campaign that explains how the shopper reached your site. The cookie is created when the JavaScript library executes, and is updated every time data is sent to Google Analytics.
 - `__utmv`: Stores visitor-level custom variable data. This cookie is created when a developer uses the `_setCustomVar` method with a visitor-level custom variable. This cookie is updated every time data is sent to Google Analytics.
 
+{:.ee-only}
+## Product Recommendations Cookies
```
Comment (leslietilling) [inline src/stores/cookie-reference.md:289]: Sentence case for sub-headings

```suggestion
## Product Recommendations cookies
```

**Review by leslietilling — APPROVED** [review]
one minor edit and good to go :)

**Comment by erikmarr** [comment]
running tests

**Pull request merged**